spec/services/nebraska_hours_calculator_spec.rb

```diff
@@ -79,7 +81,7 @@
           second_check_out = second_check_in + 10.hours + 22.minutes
           create(:attendance, child_approval: child_approval, check_in: first_check_in, check_out: first_check_out)
           create(:attendance, child_approval: child_approval, check_in: second_check_in, check_out: second_check_out)
-          expect(described_class.new(child, first_attendance_date).call).to eq(5.25 + 0.25)
+          expect(described_class.new(child, first_attendance_date).call).to eq(5.25 + 0.5)
```

Comment:
@JonErvin You deffo found a bug in the formula itself, I don't know how I got that far assuming the 0.25 was right, so I changed the formula completely. 😳